src/DuskServiceProvider.php

```diff
@@ -15,20 +15,25 @@ class DuskServiceProvider extends ServiceProvider
     public function boot()
     {
         if (! $this->app->environment('production')) {
-            Route::get('/_dusk/login/{userId}/{guard?}', [
-                'middleware' => 'web',
-                'uses' => 'Laravel\Dusk\Http\Controllers\UserController@login',
-            ]);
+            Route::group(['prefix' => env('LARAVEL_DUSK_PATH', '_dusk')], function () {
```

Comment:
This should be a config value.

Reply:
I has thought about that at the beginning, but currently where is no config file in the project. So, are you sure that I need to create config file with only 1 key?